test/Table_Tests/src/Database/Postgres_Spec.enso

```diff
@@ -261,6 +261,69 @@ postgres_specific_spec connection db_name setup =
             do_round 231 . should_be_a Decimal
             do_round 231 -1 . should_be_a Decimal
 
+    Test.group "replace: literal pattern and replacement" <|
```

Comment:
Couldn't we move these tests to `Common_Table_Operations`? We already have some `replace` tests there. We could just add flags to the `Test_Selection` specifying which combinations are supported in each backend and the test would check the results or verify that 'unsupported' error is reported. This way we will keep the tests related to the `replace` feature in one place - making them easier to maintain and to compare behaviour between backends. When they are scattered it's harder to get a bigger picture of them.

Reply:
I don't understand. Why would this test not catch it?

We should ensure that it runs on each backend, so all paths are covered. There should be no difference in coverage regardless of if the tests are put in this common file or in separate ones. What is the difference here?

Reply:
Let's say I accidentally delete an element of `supported_replace_params`. The test will decide that it should raise `Unsupported_Database_Operation`, and `Column.replace` will make that same decision, because they're both calling `are_replace_params_supported`, and the test will pass, but now one of the cases isn't implemented anymore.

I think this not a huge problem, and it's better to have one set of tests, to ensure that whatever combinations are implemented behave the same.

Reply:
> The test will decide that it should raise `Unsupported_Database_Operation`
> ...
> because they're both calling `are_replace_params_supported`

That does not seem right.

The test should not base this on `are_replace_params_supported` but it should rely on a `test_selection` or settings like that.

Reply:
There are 12 different versions of place, depending on the parameters. Are you suggesting that we have a flag for each of these versions, set differently for each dialect?